Reference/Templating/Mvc/views.md

```diff
@@ -10,82 +10,56 @@ _Working with MVC Views and Razor syntax in Umbraco_
 ```
 
-If you're using the Field method from within a partial view then be aware that you will need to pass the context so the Field method knows where to get the desired value from. For instance you can pass "CurrentPage" like this:
+If you're using the method from within a partial view then be aware that you will need to inherit the context so the method knows which type to get the desired value from. You'd do this at the top of partial view and so strongly typed properties can then be accessed in the partial view. For instance you can pass "HomePage" like this:
 
 ```csharp
-@Umbraco.Field(Model.Content, "eventLink")
+@inherits UmbracoViewPage<SiteSettings>
+...
+@Model.Value("title")
 ```
 
 You will also need to pass the "Context" to the @Umbraco.Field() method if you're looping over a selection like this where we pass the "item" variable.
 
+Lopping over a selection works in a similar way. If you have a property that contains, for instance, an IEnumberable collection, you can access the individual items using a foreach loop. Below illustrates how you might do that using "item" as a variable.
+
 ```csharp
 @{
-    var selection = Model.Content.Site().FirstChild("events").Children("event").Where(x => x.IsVisible());
+    var collection = Model.ItemList;
 }
 
 <ul>
-    @foreach(var item in selection){
-        @Umbraco.Field(item, "eventLink")
+    @foreach(var item in collection){
+        <p>@item.Name</p>
     }
 </ul>
 ```
-
-There are several optional parameters. Here is the list with their default values:
-
-* `altFieldAlias = ""`
-* `altText = ""`
-* `insertBefore = ""`
-* `insertAfter = ""`
-* `recursive = false`
-* `convertLineBreaks = false`
-* `removeParagraphTags = false`
-* `casing = RenderFieldCaseType.Unchanged`
-* `encoding = RenderFieldEncodingType.Unchanged`
-
-The easiest way to use the Field method is to specify the optional parameters you'd like to set. For example, if we want to set the insertBefore and insertAfter parameters we'd do:
-
-```csharp
-@Umbraco.Field("bodyContent", insertBefore : "<h2>", insertAfter : "</h2>")
-```
-
-## Rendering a field with Model
-
-The UmbracoHelper method provides many useful parameters to change how the value is rendered. If you however want to render value "as-is" you can use the @Model.Content property of the view. For example:
-
-```csharp
-@Model.Content.Properties["bodyContent"].Value
-```
-
-Or alternatively:
-
-```csharp
-@Model.Content.GetPropertyValue("bodyContent")
-```
-
-You can also specify the output type that you want from the property. If the property editor or value does not support the conversion then an exception will be thrown. Some examples:
-
-```csharp
-@Model.Content.GetPropertyValue<double>("amount")
-@Model.Content.GetPropertyValue<RawXElement>("xmlContents")
-```
+If you want to convert a type and it's possible, you can do that by typing a variable and assigning the value from your property to it. This might could look like the example below.
```

Comment:
```suggestion
If you want to convert a type and it's possible, you can do that by typing a variable and assigning the value from your property to it. This could look like the example below.
```